pkg/scheduler/framework/plugins/podtopologyspread/filtering.go

```diff
@@ -149,6 +149,10 @@ func (p *criticalPaths) update(tpVal string, num int) {
 
 // PreFilter invoked at the prefilter extension point.
 func (pl *PodTopologySpread) PreFilter(ctx context.Context, cycleState *framework.CycleState, pod *v1.Pod) (*framework.PreFilterResult, *framework.Status) {
+	if len(pod.Spec.TopologySpreadConstraints) == 0 && len(pl.defaultConstraints) == 0 {
```

Comment:
@Huang-Wei Thanks for your review. Since this is my first PR for K8s, I thought I would make it as small as possible.
Should I change everything at once?

Reply:
I'd highly prefer to make the logic accurate in the very beginning. While polishing logic we're usually fine with following-ups.

Reply:
👍